src/libtools/table.cc

```diff
@@ -30,12 +30,12 @@ namespace tools {
 using namespace boost;
 using namespace std;
 
-void Table::resize(int N, bool preserve) {
-  _x.resize(N, preserve);
-  _y.resize(N, preserve);
-  _flags.resize(N, preserve);
+void Table::resize(int N) {
```

Comment:
Make int const

Reply:
it is a copy of the parameter, making it const serves no purpose